### ISSUE TITLE
A context menu command to nominate users

### ISSUE THREAD

**Issue by mbaruh** [python-discord/bot#2938]
We should have a context menu command for helpers to nominate users.

* The command will be available only to helpers.
* If the helper didn't nominate the user yet, a new nomination will be added, with the nomination reason being a jump link to the message the command was invoked on.
* If there is already an existing nomination from that helper for that user, the jump link will be appended to the nomination reason.
* Either way, the helper will be pinged with an appropriate confirmation message (with whether the nomination was added or edited), similar to the ones we have already in the `!nominate` and `!tp edit reason` commands, in the `#nominations` channel.

**Comment by minalike** [comment]
With the use of this command would we still funnel the full nomination content to the `#nominations` channel?

No plan to deprecate the text `!nominate` command, correct?

**Comment by BruDriguezz** [comment]
If this issue goes ahead, I'd like to be assigned to it.

**Comment by mbaruh** [comment]
@minalike We can funnel the full nomination content in the confirmation message. Yeah, let's do that.

**Comment by mbaruh** [comment]
@BruDriguezz assigned, let me know if there are any issues, thanks!

**Comment by wookie184** [comment]
Hi @BruDriguezz, are you still interested in working on this?

**Comment by BruDriguezz** [comment]
@wookie184,
My apologies, this issue completely slipped my mind!

I actually made some initial progress on it, but unfortunately, a data corruption issue wiped out all my work.  This discouraged me a bit, and I decided to focus on other things with the intention of coming back to it later.  Unfortunately, that later never arrived, and it completely fell off my radar.

Given my current workload, I wouldn't be able to dedicate time to this issue in the foreseeable future. If you'd be interested in taking it over, I'd be absolutely happy to hand it off!
